addons/bestja_estimation_reports/models.py

```diff
@@ -89,6 +89,24 @@ class EstimationReportEntry(models.Model):
         store=True,  # Needed by graph view
         related='estimation_report.responsible_organization',
     )
+    responsible_project = fields.Many2one(
+        'bestja.project',
+        string=u"Projekt super nadrzędny",
+        store=True,  # Needed by graph view
+        related='estimation_report.responsible_project',
+    )
+    parent_project = fields.Many2one(
+        'bestja.project',
+        string=u"projekt nadrzędny",
+        related='estimation_report.parent_project',
+        store=True,
+    )
+    top_project = fields.Many2one(
+        'bestja.project',
+        string=u"projekt nadrzędny",
```

Comment:
This label is the same as for the field “parent_project". They should be different.